berserk/clients/studies.py

```diff
@@ -24,3 +24,10 @@ def export(self, study_id: str) -> Iterator[str]:
         """
         path = f"/api/study/{study_id}.pgn"
         yield from self._r.get(path, fmt=PGN, stream=True)
+
+    def get_study_by_username(
+        self, username: str
+    ) -> Generator[Dict[str, Any], None, None]:
+        # end point for api/study/by/{username}
+        path = f"/api/study/by/{username}"
+        yield from self._r.get(path, fmt=JSON, stream=True)
```

Comment:
```suggestion
        yield from self._r.get(path, fmt=NDJSON, stream=True)
```
streaming endpoints are always NDJSON or PGN